docs/index.html

```diff
@@ -369,6 +427,23 @@ <h1>Bugün</h1>
                 const logarithmicChart = new ApexCharts(document.querySelector('#logarithmicChart'), logarithmicOptions);
                 logarithmicChart.render();
 
+                //logarithmic new patients
+                var matchedList = $.map(values, function (n, i) {
```

Comment:
Array.map metodu kullanılabilir diğer taraflarda da onu kullandık bütünlük oluşması için değiştirilebilir.